test/e2e/validation_test.go

```diff
@@ -153,7 +165,24 @@ var _ = Describe("Validation", func() {
 	})
 
 	It("should validate the communication matrix ports match the node's listening ports", func() {
-		err := commatrix.WriteMatrixToFileByType(utilsHelpers, "communication-matrix", types.FormatCSV, deployment, artifactsDir)
+		val, exists := os.LookupEnv("ENDPOINT_SLICE_EXTRA_OPEN_PORTS_FILE")
```

Comment:
why not to do this on the before each?
will not need it on the nftables also? maybe we want to add ENDPOINT_SLICE_EXTRA_OPEN_PORTS_FORMAT

Reply:
Those ports are open ports that does not have endpoint slices, so we only want to add them to the matrix in the test comparing endpoint slices and ss matrices - to avoid unnecessary failure.
In the other tests it will be meaning less.